packages/@webex/media-helpers/package.json

```diff
@@ -14,7 +14,8 @@
     "node": ">=16"
   },
   "dependencies": {
-    "@webex/internal-media-core": "1.38.0"
+    "@webex/internal-media-core": "1.38.0",
+    "@webex/web-media-effects": "^2.8.0"
```

Comment:
Do we need to use ^ icon before the version here ? We follow this practice in calling SDK and web client where do not use caret sign because it allows to use the most recent Major(first number) version. That means it can use any latest 2.x.x release of media-effects I I believe

Reply:
In JS-SDK I don't think so we follow this convention of not using caret sign. AFAIK, by using caret sign we're saying that accept any minor and patch version update for this package which is acceptable as it won't break our code most of the times and we will be using latest minor or patch released version.